client/extensions/woocommerce/app/products/products-list-pagination.js

```diff
@@ -20,9 +21,7 @@ const ProductsListPagination = ( {
 	requestedPage,
 	onSwitchPage,
 } ) => {
-	const perPage = 10;
-
-	if ( totalProducts && totalProducts < perPage + 1 ) {
+	if ( totalProducts && totalProducts < DEFAULT_QUERY.per_page + 1 ) {
```

Comment:
Are we sure we want `perPage` hard coded down in `state/sites/` instead of `app/`?

Reply:
I think it makes more sense to hard code it in one place (`DEFAULT_QUERY`) vs more than one (in the request _and_ here in the pagination). Otherwise it would be nice to get the value from the request directly, but I don't think we change per_page.